Tests/CommonMarkTests/NodeHashableTests.swift

```diff
@@ -0,0 +1,64 @@
+import XCTest
+import CommonMark
+
+final class NodeHashableTests: XCTestCase {
+    func testNodesEquality() {
+        let firstNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+        let secondNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+
+        XCTAssertNotEqual(firstNode, secondNode)
+        XCTAssertEqual(firstNode, firstNode)
+        XCTAssertEqual(secondNode, secondNode)
+    }
+
+    func testNodesHashable() {
+        let firstNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+        let secondNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
```

Comment:
Is there any reason not to assign the same text into a common variable? If the concern has to do with initialization from a single value, we could achieve the same effect with string interpolation:

```suggestion
        let text = #"""
        All human beings are born free and equal in dignity and rights.
        They are endowed with reason and conscience
        and should act towards one another in a spirit of brotherhood.
        """#
        
        let firstNode = Paragraph(text: "\(text)", replacingNewLinesWithBreaks: true)
        let secondNode = Paragraph(text: "\(text)", replacingNewLinesWithBreaks: true)
```